client/my-sites/domains/controller.jsx

```diff
@@ -248,15 +248,15 @@ const redirectIfNoSite = redirectTo => {
 	};
 };
 
-const redirectToAddMappingIfVipSite = () => {
+const redirectToUseYourDomainIfVipSite = () => {
 	return ( context, next ) => {
 		const state = context.store.getState();
 		const selectedSite = getSelectedSite( state );
 		const domain = context.params.domain ? `/${ context.params.domain }` : '';
-		const query = stringify( { initialQuery: context.params.suggestion } );
+		const query = stringify( { initialQuery: get( context, 'params.suggestion', '' ) } );
```

Comment:
Apparently, if we omit `initialQuery` (because `context.params.suggestion` is missing - which is semi-normal), we trigger a regression that crept in (we expect `initialQuery` param to always be present).

Reply:
Good catch!